config-connector/solutions/sql/kpt/mysql-ha/README.md

```diff
@@ -0,0 +1,47 @@
+MySQL High Availability
+==================================================
+# NAME
+  mysql-ha
+# SYNOPSIS
+  Config Connector compatible YAMLs for creating a high availability MySQL cluster
+# CONSUMPTION
+  Download the package using [kpt](https://googlecontainertools.github.io/kpt/).
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/sql/kpt/mysql-ha mysql-ha
+  ```
+# REQUIREMENTS
+  A working Config Connector installation configured to manage a project with Cloud SQL Admin API enabled.
+# SETTERS
+|     NAME      |       VALUE       |     SET BY      |          DESCRIPTION          | COUNT |
+|---------------|-------------------|-----------------|-------------------------------|-------|
+| database-name | mysql-ha-database | package-default | name of SQL database          | 1     |
+| instance-name | mysql-ha-instance | package-default | name of SQL instance          | 5     |
+| password      | ${PASSWORD?}      | PLACEHOLDER     | SQL password (base64 encoded) | 1     |
+| region        | us-central1       | package-default | region of SQL instance        | 2     |
+| username      | ${USERNAME?}      | PLACEHOLDER     | name of SQL user              | 1     |
+| zone          | us-central1-b     | package-default | zone of instance              | 1     |
+| zone-backup   | us-central1-c     | package-default | zone of backup instance       | 1     |
+# USAGE
+  Configure setters using kpt as follows:
+  ```
+  kpt cfg set . NAME VALUE
+  ```
+  Setting placeholder values is required, changing package-defaults is optional.
+
+  Set `username` to the SQL username that you will use to access the database.
+  ```
+  kpt cfg set . username your-username
+  ```
+  _Optionally_ set `database-name`, `instance-name`, `region`, `zone`, and
+`zone-backup` in the same manner. Note that if your instance is deleted the
+name you used will be reserved for 7 days. You will need to use a new name in
+order to re-create the instance.
```

Comment:
The message should be more noticeable like below:
````
 _Optionally_ set `database-name`, `instance-name`, `region`, `zone`, and
`zone-backup` in the same manner. 

**Note:** If your SQL Instance is deleted, the name you used will be reserved
for **7 days**. In order to re-apply this solution, you need to run
`kpt cfg set . instance-name new-instance-name` to change to a new
instance name that hasn't been used in the last 7 days.
````

Reply:
Done.